src/methods/validate-input-query-fixture-match.ts

```diff
@@ -74,15 +77,32 @@ export async function validateInputQueryFixtureMatch(
 
 /**
  * Remove query names from GraphQL strings for comparison
- * 
+ *
  * GraphQL's print() function:
- * - Named queries: "query SomeName { ... }" prints as "query SomeName { ... }"  
+ * - Named queries: "query SomeName { ... }" prints as "query SomeName { ... }"
  * - Anonymous queries: "query { ... }" prints as "{ ... }" (drops query keyword)
  * - convertFixtureToQuery generates anonymous queries that print as "{ ... }"
- * 
+ *
  * We only need to remove query names - anonymous queries are already in the right format!
  */
 function removeQueryName(graphqlString: string): string {
   return graphqlString
     .replace(/^query\s+\w+\s*/, '');  // Remove named queries: "query SomeName {...}" → "{...}"
-}
\ No newline at end of file
+}
+
+/**
+ * Remove field aliases from GraphQL query strings
+ *
+ * Aliases like "itemList: items" are normalized to just "items"
+ * This is needed because convertFixtureToQuery generates queries with actual field names (no aliases)
+ *
+ * Uses a negative lookahead to avoid matching colons inside:
+ * - String literals: "value: \"test\""
+ * - Field arguments: field(key: "value")
+ */
+function removeAliases(graphqlString: string): string {
+  // Match alias pattern: word followed by colon and whitespace and another word
+  // But only when followed by whitespace, opening brace, or closing brace
+  // This avoids matching colons inside strings or arguments
+  return graphqlString.replace(/\b(\w+):\s*(\w+)(?=[\s{},])/g, '$2');
```

Comment:
This feels like it has the potential to miss a number of edge cases. I would much prefer if we could manipulate the AST instead of doing regex operations.

Additionally, how do we expect to handle the case of the same field selected multiple times with aliases? For example:
```graphql
{
  discountNode {
    metafield1: metafield(key: "my-key1", namespace: "my-namespace") { jsonValue }
    metafield2: metafield(key: "my-key2", namespace: "my-namespace") { jsonValue }
  }
}
```